README.md

```diff
@@ -55,6 +55,16 @@ The project has the following prerequisites:
 * Ruby (for running the documentation website locally)
 * Docker (for running the visual regression testing framework)
 
+### Few differences between Fundamental Library Styles and [Fundamental](https://github.com/SAP/fundamental)
+
+Fundamental Library Styles is aiming to deliver:
+* [Fiori 3.0](https://www.sap.com/products/fiori.html) compliant components
+* themable components built on top of [SAP Theming Base Content](https://github.com/SAP/theming-base-content) by consuming the CSS Custom Properties delivered by the theming library
+* self-contained components(each component style file contains all the styling needed to be rendered properly and at the same time avoid external styling bleeding in)
+* good accessibility support
```

Comment:
This is vague.

* Accessibility - Color contrast support for WCAG 2.0 level AA (4.5:1 for typical text)
* Accessibility - Semantic HTML reference
* Accessibility - Aria attributes noted when possible in HTML reference